api/src/exam-environment/utils/exam.ts

```diff
@@ -7,22 +7,33 @@ import {
   EnvGeneratedExam,
   EnvMultipleChoiceQuestion,
   EnvQuestionSet,
-  EnvQuestionSetAttempt,
-  user
+  EnvQuestionSetAttempt
 } from '@prisma/client';
 import { type Static } from '@fastify/type-provider-typebox';
 import * as schemas from '../schemas';
 
+interface CompletedChallengeId {
+  completedChallenges: {
+    id: string;
+  }[];
+}
```

Comment:
```suggestion
type CompletedChallenges = {
  id: string;
}[];
```
To go with the suggestion below.